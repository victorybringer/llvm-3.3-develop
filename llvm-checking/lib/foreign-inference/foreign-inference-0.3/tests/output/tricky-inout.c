int f(int *i, int x) {
  if(x > 10)
    *i = 100;

  return *i + 1;
}
