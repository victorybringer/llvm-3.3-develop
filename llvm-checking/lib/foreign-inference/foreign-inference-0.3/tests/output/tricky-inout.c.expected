fromList [ ("@f", fromList [("%i", PAInOut)]) ]
